fellow-submissions/devonte-duncan.md

```diff
@@ -0,0 +1,154 @@
+
+    var a = 1;
+    ```
+    The above code will log:
+        "undefined"
+    
+    This logs "undefined" because even though the declaration of the variable `a` is hoisted
+    before runtime, while its initialization is not. Thus the value of `a` on line 60 is
+    logged as "undefined", until it assigned a value of 1 on line 62.
+    
+  
+**5. What does the following code log? Why?**
+    ```javascript
+    console.log(a);
+
+    let a = 1;
+    ```
+    This logs an error, because `let` is a scoped variable, JavaScript cannot access it until 
+    it's been initialized. Therefore, even though it's declaration is hoisted, its initialization
+    must also occur before it's referenced in order to work.
+
+**6. In the code snippet below, the function declaration comes _after_ the variable declaration in the code below. Why does it still work?**
+    ```javascript
+    let name = "Devonte";
+
+    function sayItLoud() {
+      console.log(`${name}!!!`)
+    }
+
+    sayItLoud();
+    ```
+    The variable `name` was declared and initialized within the global scope, and 
+    therefore can be seen/accessed from all other scopes as well.
+    
+**7. Why does the following code throw an error? What is the key difference between it and the example above?**
+    ```javascript
+    sayItLoud();
+
+    function sayItLoud() {
+      console.log(`${greeting}!!!}`)
+    }
+
+    let greeting = "Hello"
+    ```
+    This logs an error, because `let` is a scoped variable, JavaScript cannot access it until 
+    it's been initialized. Therefore, even though it's declaration is hoisted and it's declared in 
+    the global scope, its initialization must also occur before it's referenced in order to work.
+
+**8. What does the following code log? Why?**
+    ```javascript
+    let problemSet;
+
+    function complete(assignment) {
+      assignment = "done";
+    }
+
+    complete(problemSet);
+    console.log(`Status for Problem Set: ${problemSet}`);
+    ```
+    
+    This code logs "Status for Problem Set: undefined". This is because the variable
+    `problemSet` is declared globally, and then initialized locally within the `complete` 
+    function. Therefore, the console log on line 118 has no access to the value assigned 
+    on line 114.
+
+**9. Why does this code throw an error?** 
+    ```javascript
+    const isSunny🌞 = true;
+
+    if (isSunny🌞) {
+      let advice = 'Wear your sunshades!';
+    }
+
+    console.log(advice);
+    ```
+   The variable `advice` is declared locally within the if statement, therefore when 
+   trying to log its value on line 134, JavaScript logs an error because its existence 
+   is unknown globally.
+
+**10. So... why does this code work as intended?**
+    ```javascript
+    const isRainy🌧 = true;
+
+    if (isRainy🌧) {
+      var advice = 'Pack your umbrella.';
+    }
+
+    console.log(advice);
+    ```
+    
+    This code works despite the variable `advice` being declared locally because
```

Comment:
Half correct. Make this change.

Variables declared with `var` are function scoped, not block scoped. Thus, `advice` can be accessed outside of the block where it is declared.